pkg/parsers/kernel/kernel.go

```diff
@@ -1,11 +1,10 @@
-// +build !windows
+// +build darwin linux freebsd
```

Comment:
This should either be 
// +build darwin linux freebsd solaris
OR 
// +build !windows